apps/ledger-live-desktop/src/renderer/Default.tsx

```diff
@@ -295,11 +295,7 @@ export default function Default() {
                             <Route path="/card" render={withSuspense(Card)} />
                             <Redirect from="/manager/reload" to="/manager" />
                             <Route path="/manager" render={withSuspense(Manager)} />
-                            <Route
-                              path="/platform"
-                              component={() => <PlatformCatalog db={discoverDB} />}
-                              exact
```

Comment:
The `exact` prop is needed here and breaks the behaviour for the LiveApp component, you need to test more than just the component you touch 😉